proto/viam/app/v1/app.proto

```diff
@@ -441,6 +444,18 @@ message GetRobotRequest {
   string id = 1;
 }
 
+message GetRoverRentalRobotsRequest {}
+
+message RoverRentalRobot {
+  string robot_id = 1;
+  string location_id = 3;
+  string robot_name = 4;
```

Comment:
Should this be = 3 ?

Reply:
ah yeah, removed a field and forgot to update the numbers :/ re-pushed